R/board_url.R

```diff
@@ -139,6 +139,12 @@ pin_store.pins_board_url <- function(board, name, paths, metadata,
   abort_board_read_only("board_url")
 }
 
+#' @rdname board_deparse
+#' @export
+board_deparse.pins_board_url <- function(board, ...) {
```

Comment:
There's no reason this couldn't be deparsed, although I don't think you need to implement it, so I think you can just rely on the default behaviour here.